website/layouts/partials/footer.html

```
@@ -8,7 +8,7 @@ <h4 class="footer__heading">Documentation</h4>
           <li><a href="{{ "docs/overview/" | relURL }}">Overview</a></li>
           <li><a href="{{ "docs/installation/" | relURL }}">Installation</a></li>
           <li><a href="{{ "docs/quickstart/" | relURL }}">Quickstart</a></li>
-          <li><a href="{{ "docs/architecture/" | relURL }}">Architecture</a></li>
+          <li><a href="{{ "docs/architecture" | relURL }}">Architecture</a></li>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)